salt/example-critical-css.top

```diff
@@ -8,10 +8,6 @@ base:
         - elife.nginx-error-pages
         - elife.nodejs6
         - elife.redis-server
-        {% if pillar.elife.env in ['ci', 'dev'] %}
-        - elife.java
-        - elife.selenium
-        {% endif %}
         - journal.curl-7-36
         - journal
         - elife.proofreader-php
```

Comment:
Can lose ones like this too.

Reply:
`proofreader-php` should be able to. `journal.tenon` too.